packages/keyring-eth-ledger-bridge/src/ledger-mobile-bridge.test.ts

```diff
@@ -220,8 +224,8 @@ describe('LedgerMobileBridge', function () {
       );
     });
 
-    it('throws error when transport.deviceMode is not set', async function () {
-      mockTransport.deviceModel = null;
+    it('throws error when transport.deviceModel is not set', async function () {
+      mockTransport.deviceModel = null as unknown as MockDeviceModel;
```

Comment:
Instead of typecasting, do you think it's better to just add a directive to ignore the error?

```suggestion
    it('throws error when transport.deviceModel is not set', async function () {
      // // @ts-ignore deviceModel must be null to validate test case
      mockTransport.deviceModel = null;
```

Reply:
I think I prefer your suggestion indeed, lemme change that!